docs/sources/tempo/release-notes/v2-5.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Version 2.5 release notes
+menuTitle: V2.5
+description: Release notes for Grafana Tempo 2.5
+weight: 35
+---
+
+# Version 2.5 release notes
+
+The Tempo team is pleased to announce the release of Tempo 2.5.
+
+This release gives you:
+
+* New features, including more TraceQL metrics, gRPC streaming endpoints, and new experimental vParquet4 block format
+* TraceQL enhancements and performance improvements
+* Performance and stability enhancements
+
+As part of this release, we’ve updated ownership for `/var/tempo` from `root:root` to a new `tempo:tempo` user with a UUID of `10001`. Learn about this breaking change in Upgrade considerations.
+
+Read the [Tempo 2.5 blog post](/blog/2024/02/29/grafana-tempo-2.4-release-traceql-metrics-tiered-caching-and-tco-improvements/) for more examples and details about these improvements.
+
+These release notes highlight the most important features and bugfixes. For a complete list, refer to the [Tempo changelog](https://github.com/grafana/tempo/releases).
+
+{{< youtube id="c4gW9fwkLhc" >}}
+
+## Features and enhancements
+
+The most important features and enhancements in Tempo 2.5 are highlighted below.
+### Additional TraceQL metrics (experimental)
+
+In this release, we’ve added several [TraceQL metrics](https://grafana.com/docs/tempo/next/operations/traceql-metrics/). Tempo 2.4 introduced the `rate()` function to view rates of spans. For this release, we’ve added `quantile_over_time` and `histogram_over_time`. [PR 3605](https://github.com/grafana/tempo/pull/3605), [PR 3633](https://github.com/grafana/tempo/pull/3633), [PR 3644](https://github.com/grafana/tempo/pull/3644)]
+
+You can use allowing users to aggregate numerical values, such as the all-important span duration. Notice that you can specify multiple quantiles in the same query.
```

Comment:
You can use `quantile_over_time` to aggregate numerical values
?

Reply:
Good catch!